### PR TITLE
[2.14.x] Fixes bug where installer would overwrite the solr password

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Fixes an issue where the installer would overwrite the solr password after it had already been set earlier on in the install process. 

#### Who is reviewing it? 
@ahoffer @Kjames5269 @jhunzik 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@emmberk

#### How should this be tested?
Run through the installer and hit restart now at the end.
Open DDF and verify that Intrigue starts up. 
Look at custom.system.properties and verify that the solr.password is some encrypted password.

#### Any background context you want to provide?
This PR (https://github.com/codice/ddf/pull/4429) introduced the issue by frontloading the read of the system properties file which makes it fall out of sync after the solr properties are updated.
I've refactored it to read every time these methods are called to ensure it doesn't fall out of sync.

#### What are the relevant tickets?
Fixes: #4570

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by tyler30clemens** [comment]
Hero steps Successful ✅

Installed through the UI and hit the restart 
- Verified Solr password was encrypted in custom.system.properties
- Verified able to hit Intrigue as well as query and ingest 
- Verified manual shutdown and restart worked with no issues

File: platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java [hunk 1]
```diff
@@ -135,72 +128,78 @@ public SystemPropertiesAdmin(GuestClaimsHandlerExt guestClaimsHandlerExt)
...
-            SITE_NAME_TITLE,
-            SITE_NAME_DESCRIPTION,
-            null,
-            systemDotProperties));
-    properties.add(
-        getSystemPropertyDetails(
-            SystemInfo.VERSION, VERSION_TITLE, VERSION_DESCRIPTION, null, systemDotProperties));
+    Properties systemDotProperties = null;
+    try {
+      systemDotProperties = new Properties(systemPropertiesFile);
```
Comment (emmberk) [inline platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java]: ✏️ Not a blocker for this PR: I don't like that we have to read this file twice. Once when we populate the installer page and then another time when we save the installer page. Are we worried about the system.properties being changed in between those 2 reads? Can we rely on the system properties instead to populate the installer page and only ready the system.properties file in order to overwrite the few values. Or, can we use the same read for the corresponding write?

File: platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java [hunk 2]
```diff
@@ -122,11 +120,6 @@ public SystemPropertiesAdmin(GuestClaimsHandlerExt guestClaimsHandlerExt)
     super(SystemPropertiesAdminMBean.class);
     this.guestClaimsHandlerExt = guestClaimsHandlerExt;
     configureMBean();
-    try {
-      systemDotProperties = new Properties(systemPropertiesFile);
```
Comment (emmberk) [inline platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java]: ✏️ I think we should add a unit test like

1. instantiate this class
2. override one value in system.properties and set the system property (might want to try both a property than can be changed in the installer page and one that isn't, like the solr property)
3. `readSystemProperties`
4. `writeSystemProperties`

**Comment by vinamartin** [comment]
build now

**Pull request merged**